src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/WorkloadManifest.cs

```diff
@@ -10,6 +10,8 @@ namespace Microsoft.NET.Sdk.WorkloadManifestReader
     /// </summary>
     public class WorkloadManifest
     {
+        public WorkloadManifest(string id) : this(id, new FXVersion(7, 3, 5), null, string.Empty, [], [], []) { }
+
```

Comment:
This looks like it's just for tests, but that wouldn't be obvious if you were trying to create a `WorkloadManifest` in the product code.  Could you do one of the following?

- Create a helper method in the tests that creates a WorkloadManifest with some default values given an ID?
- Instead of a constructor, have a static `CreateForTests` factory method on the `WorkloadManifest` class so that it's clear what the purpose of the method is.

Reply:
Good point. I like the idea of the static CreateForTests factory method; will implement that.